operator/operator.yaml

```diff
@@ -16,6 +16,7 @@ tasks:
       - cassandra-env-sh.yaml
       - jvm-options.yaml
       - stateful-set.yaml
+      - cassandra-readiness-probe-sh.yaml
```

Comment:
You're using `readiness_probe` here and `readiness_check` in other places. Let's use one or the other? Since we're in the world of k8s, let's go with `probe`, WDYT?

Reply:
> You're using `readiness_probe` here and `readiness_check` in other places. Let's use one or the other? Since we're in the world of k8s, let's go with `probe`, WDYT?

Make sense. Thanks!